src/Helper/HeadScript.php

```diff
@@ -401,7 +406,10 @@ public function itemToString($item, $indent, $escapeStart, $escapeEnd)
         $addScriptEscape = ! (isset($item->attributes['noescape'])
             && filter_var($item->attributes['noescape'], FILTER_VALIDATE_BOOLEAN));
 
-        if (empty($item->type) && $this->view && $this->view->plugin('doctype')->isHtml5()) {
+        if ((empty($item->type) || $item->type === self::DEFAULT_SCRIPT_TYPE)
```

Comment:
I think check should be case insensitive, so we should use `strtolower($item->type)` instead. What do you think?

Reply:
Added strtolower and a new assertion to the unit test